src/test/java/uk/nhs/hee/trainee/details/service/ProgrammeMembershipServiceTest.java

```diff
@@ -499,6 +510,303 @@ void shouldNotSignCojWhenTraineeFoundButProgrammeMembershipsNotExists() {
     verify(repository, never()).save(traineeProfile);
   }
 
+  @Test
+  void newStarterShouldBeFalseIfTraineeNotFound() {
+    when(repository.findByTraineeTisId(TRAINEE_TIS_ID)).thenReturn(null);
+
+    boolean isNewStarter = service.isNewStarter(TRAINEE_TIS_ID, PROGRAMME_TIS_ID);
+
+    assertThat("Unexpected isNewStarter value.", isNewStarter, is(false));
+  }
+
+  @Test
+  void newStarterShouldBeFalseIfPmNotFound() {
+    TraineeProfile traineeProfile = new TraineeProfile();
+    traineeProfile.setProgrammeMemberships(
+        List.of(getProgrammeMembershipDefault("unknown id",
+            PROGRAMME_MEMBERSHIP_TYPE, START_DATE, END_DATE))); //PROGRAMME_TIS_ID != "unknown id"
+    when(repository.findByTraineeTisId(TRAINEE_TIS_ID)).thenReturn(traineeProfile);
+
+    boolean isNewStarter = service.isNewStarter(TRAINEE_TIS_ID, PROGRAMME_TIS_ID);
+
+    assertThat("Unexpected isNewStarter value.", isNewStarter, is(false));
+  }
+
+  @ParameterizedTest
+  @NullSource
+  @MethodSource("listNonNewStartPmTypes")
+  void newStarterShouldBeFalseIfPmHasWrongType(String pmType) {
+    TraineeProfile traineeProfile = new TraineeProfile();
+    traineeProfile.setProgrammeMemberships(
+        List.of(getProgrammeMembershipDefault(PROGRAMME_TIS_ID, pmType, START_DATE, END_DATE)));
+    when(repository.findByTraineeTisId(TRAINEE_TIS_ID)).thenReturn(traineeProfile);
+
+    boolean isNewStarter = service.isNewStarter(TRAINEE_TIS_ID, PROGRAMME_TIS_ID);
+
+    assertThat("Unexpected isNewStarter value.", isNewStarter, is(false));
+  }
+
+  @ParameterizedTest
+  @NullSource
+  @CsvSource({"1970-01-01"})
```

Comment:
You could use `@ValueSource(strings = "1970-01-01")` here, which doesn't make much difference but does save an import.